CONTRIBUTING.md

```diff
@@ -1,32 +1,41 @@
-# Contributig to Arcus
+# Contributing to Arcus
 🎉 First off, thanks for taking the time to contribute! We're really glad you're reading this. 🎉
 
-The following set of guidelines will help you in contributing to  Arcus repositories.
+The following set of guidelines will help you in contributing to Arcus repositories.
 But, no worries, if you forget something or have problems with some of these steps, we're happy to help! In those cases: create a PullRequest on the repository so we can talk about it.
 
 This contribution guide has the following contents:
 
 ## Issues and discussions
 We are happy to discuss ideas and possible features or problems.
 
-In these cases, we would like you to create an issue:
-- You encouter a problem or bug while using Arcus.
-- You have a clear idea for a feature or enhancement in Arcus.
-
-In these cases, we should like you to create a discussion:
+**In these cases, we would like you to create a discussion**
 - You want to discuss an raw idea with us
 - You have a question on the library, how to use it or where to find something
 - You want to raise a concern
 
-Work will be assigned by the code owners of the Arcus library and it is recommended to check with them first diving into code.
-> Also: a good tip is to assign one or every code owner of the repository to your PullRequest.
+**In these cases, we would like you to create an issue**
+- You encounter a problem or bug while using Arcus.
+- You have a clear idea for a feature or enhancement in Arcus.
+
+After you have created an issue, work will will be assigned by the code owners of the Arcus library.  However, nothing you are also encouraged to fix the bug you've reported or implement the feature that you've proposed yourself as well!  In that case, it is recommended to check with the code owners of the Arcus repository first before diving into code.
```

Comment:
"nothing stops you" probably?

Reply:
Yes, I was first going to write 'nothing stops you', but then decided to rephrase.  I'll modify this.